fast_api_als/routers/submit_lead.py

```diff
@@ -23,6 +23,19 @@
 
 router = APIRouter()
 
+logger = logging.getLogger(__name__)
```

Comment:
Missing logs in some of the decisions.
Didn't see any logs related to time taken.